modules/ROOT/pages/gov-cloud-features.adoc

```diff
@@ -6,6 +6,17 @@ endif::[]
 MuleSoft Government Cloud includes the full functionality of the Anypoint Platform runtime plane (services and Mule runtime engine) and a subset of the Anypoint Platform control plane (browser-based management of APIs and Mule runtime engine instances) functionality. Government Cloud supports the full API-led connectivity methodology and end-to-end API lifecycle management.
 
 MuleSoft Government Cloud supports Mule runtime engine v3.9 and later.
+[IMPORTANT]
+====
+Because MuleSoft Government Cloud does not support Mule agents with self-signed certificates, specific versions of Mule runtime engine are required:
+
+* Mule 3: v1.12.0 and later
```

Comment:
We will need to remove this line also because we just support mule 4 in gov envs.

Reply:
Is this true for all GovCloud runtimes? (both cloud and on-prem)?

Reply:
@bricker386 this only apply for `on-prem`

Reply:
ok, so we shouldn't remove this from the general GovCloud runtime support. It should be called out as specific to on-prem.

Reply:
Added some text.